canonical-web-and-design/ubuntu.com
# Move payment methods page to global nav

## Done

- Moved /advantage/payment-methods to /account/payment-methods
- Added a link to the new page in the nav
- added a redirect from the old to the new
- Updated links to the old page

## QA

- login https://ubuntu-com-10003.demos.haus/account/payment-methods
- go to /advantage/payment-methods, it should redirect you to /account/payment-methods
- check that the link appears correctly under your name in the nav
- check that the page is still the same


## Screenshots

![image](https://user-images.githubusercontent.com/11927929/124944237-70681c80-e00d-11eb-8cba-c8be12a31d68.png)
